### PR TITLE
feature/upgrade-to-archetype

### PR DESCRIPTION
**Pull request by DariuszBlicharski**
Upgrade to adapt-service-without-webapi to use archetype

### CODE REVIEW

File: src/render/blog/adapt-service-without-webapi.html.md [hunk 1]
```diff
@@ -220,69 +247,101 @@ public class BooksDbAdapterConfiguration {
...
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import rx.Single;
+
+public class ExampleServiceAdapterProxy extends AbstractAdapterProxy {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(ExampleServiceAdapterProxy.class);
+
+	@Override
+	protected Single<AdapterResponse> processRequest(AdapterRequest adapterRequest) {
```
Comment (tomaszmichalak) [inline src/render/blog/adapt-service-without-webapi.html.md]: Please check code formatting in this snippet.

File: src/render/blog/adapt-service-without-webapi.html.md [hunk 2]
```diff
@@ -96,37 +96,46 @@ Page markup will look like following snippet:
...
 ## Set up the project
 
-We will show you how to configure a custom adapter project using _Maven_ - feel free to use any other favourite
+We will show you how to create a custom adapter project using _Maven_ archetype - feel free to use any other favourite
 project build tool. To build and run this tutorial code you need _Java 8_ and _Maven_.
 
-Create a `pom.xml` file with dependencies on `knotx-core` and `knotx-adapter-common`. Additional dependencies 
-we will use are [`vertx-jdbc-client`](http://vertx.io/docs/vertx-jdbc-client/java/) and 
-`hsqldb` driver. The `<dependencies>` section of your project's `pom.xml` should look like this:
+Follow the instructions from [`here`](https://github.com/Knotx/knotx-extension-archetype) to create project
```
Comment (DariuszBlicharski) [inline src/render/blog/adapt-service-without-webapi.html.md]: fixed

File: src/render/blog/adapt-service-without-webapi.html.md [hunk 3]
```diff
@@ -140,74 +148,93 @@ class provided by RXJava _Vert.x_.
...
 import io.vertx.core.Context;
 import io.vertx.core.Vertx;
+import io.vertx.core.eventbus.MessageConsumer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
 import io.vertx.rxjava.core.AbstractVerticle;
+import io.vertx.serviceproxy.ProxyHelper;
+
+public class ExampleServiceAdapter extends AbstractVerticle {
```
Comment (DariuszBlicharski) [inline src/render/blog/adapt-service-without-webapi.html.md]: This is current version (it is a file created by adapter without our changes).

File: src/render/blog/adapt-service-without-webapi.html.md (hunk 1 above)
Comment (DariuszBlicharski) [inline src/render/blog/adapt-service-without-webapi.html.md]: fixed

File: src/render/blog/adapt-service-without-webapi.html.md [hunk 4]
```diff
@@ -96,37 +96,46 @@ Page markup will look like following snippet:
...
+but we used these in tutorial: 
+1. groupId: `io.knotx.tutorial`
+2. artifactId: `custom-service-adapter`
+3. version: `1.1.1`
+4. package name: `io.knotx.tutorial`
+5. project name: `First custom service adapter`
+
+Created `pom.xml` file will have dependencies on `knotx-core` and `knotx-adapter-common`
+(There are also other dependencies, but for the purpose of this exercise we need only those two). 
+Additional dependencies we will use are [`vertx-jdbc-client`](http://vertx.io/docs/vertx-jdbc-client/java/) and
```
Comment (tomaszmichalak) [inline src/render/blog/adapt-service-without-webapi.html.md]: Additional dependencies we will use are
->
Additional we will use also

File: src/render/blog/adapt-service-without-webapi.html.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tomaszmichalak) [reply src/render/blog/adapt-service-without-webapi.html.md]: OK

**Pull request merged**